nibabel/gifti/gifti.py

```diff
@@ -68,10 +68,10 @@ def print_summary(self):
 
 
 class GiftiNVPairs(object):
-
+    """
     name = str
     value = str
-
+    """
```

Comment:
For these, if possible, I'd rather see proper docstrings than simply commenting these out.

At simplest, a quick description and a type description, e.g.:

    class GiftiNVPairs(object):
        """Name-value pair objects

        Instance variables
        ------------------
        name    : str
        value   : str
        """

Documenting instance variables doesn't seem to be standard nibabel practice, so I'm also open to just removing these. Still, seems a little bit of a shame to lose the type information altogether. Perhaps @matthew-brett has a feeling on best direction to go with these?